client/layout/universal-navbar-header/nav-style.scss

```diff
@@ -28,6 +28,21 @@ $studio-blue-30: #0675c4;
 	}
 }
 
+.is-section-themes,
+.is-section-theme {
+	&.has-no-sidebar {
+		.masterbar {
+			height: 56px;
+			position: relative;
```

Comment:
In regard to this change, the masterbar in production will remain fixed, together with the search form and filters when the user scrolls down. WIth this change, the masterbar will no longer be fixed, and will be scrolled up. Is that expected?

Reply:
@tyxla - yes, this is similar to the behaviour in logged out /plugins